.gitlab-ci.yml

```diff
@@ -1,14 +1,13 @@
-image: registry.gitlab.com/pcengines/apu2-documentation:latest
+image: pcengines/pce-fw-builder-legacy:latest
 
 services:
   - docker:dind
 
 variables:
-  RELEASE_DIR: ${CI_PROJECT_DIR}/apu2_fw_rel
+  RELEASE_DIR: /home/coreboot/release
```

Comment:
Release directory should be CI_PROJECT_DIR to ensure artifacts will be available across jobs